### PR TITLE
Add info to the README.md and CONTRIBUTING.md files

### PR DESCRIPTION
**Pull request by lotif**
# PR Type
Documentation

# Short Description

Clickup Ticket(s): https://app.clickup.com/t/86877aha9

Adding information about building and running the project into readme and contributing files.

# Tests Added
NA

### CODE REVIEW

**Comment by lotif** [comment]
@jewelltaylor I've added some information about buiding and running what we have so far in the https://github.com/VectorInstitute/FLorist/pull/3 PR. Let me know if there is more stuff you'd like to see there.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+```shell
+virtualenv venv
+source venv/bin/activate
+```
+
+We use [Poetry](https://python-poetry.org/) to manage back-end dependencies:
+
+```shell
+pip install --upgrade pip poetry
+poetry install
```
Comment (jewelltaylor) [inline README.md:21]: I got a strange looking warning when running this, not sure if you ran into the same thing and just wanted to flag it. It seems to be in regards to poetry expecting a certain format to the directory. This [issue](https://github.com/python-poetry/poetry/issues/8662) seems to describe it. I guess it comes down to using poetry for dependency management vs poetry for dependency management and packaging. I guess we are only going to be using for dependency management  so this doesn't seem relevant.

Comment (lotif) [reply README.md:21]: Thanks for catching that. I just ignored the warning but reading that issue, poetry expects to have a folder with the package name so I've changed #3 to reflect that and the warning doesn't appear anymore.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -13,6 +13,35 @@ Once the python virtual environment is setup, you can run pre-commit hooks using
 pre-commit run --all-files
 ```
 
+## Development dependencies
+
+To install development dependencies, first you need to create a virtual environment.
+The easiest way is by using the [virtualenv](https://pypi.org/project/virtualenv/) package:
```
Comment (jewelltaylor) [inline CONTRIBUTING.md:19]: Should we explicitly mention we expect python between 3.9 and 3.11? Or it is it enough to have it in the pyproject.toml

**Review by jewelltaylor — APPROVED** [review]
Looks good to me! I was able to get the front end client and back end api up and running locally with these instructions, so LGTM!

File: CONTRIBUTING.md (hunk 2 above)
Comment (lotif) [reply CONTRIBUTING.md:19]: I think having it in `pyproject.toml` is enough, we don't have a hard requirement on it ourselves. Besides, when we change it, it's gonna be one less thing for us to worry about.

**Pull request merged**